lib/Doctrine/Common/Lexer.php

```diff
@@ -13,6 +13,8 @@
  * @author Guilherme Blanco <guilhermeblanco@hotmail.com>
  * @author Jonathan Wage <jonwage@gmail.com>
  * @author Roman Borschel <roman@code-factory.org>
+ *
+ * @deprecated Use Doctrine\Common\Lexer\AbstractLexer from doctrine/lexer package instead.
```

Comment:
Do we even want to have a `doctrine/lexer`? I thought the @hoa stuff would fix this, once their versioning system works too...

Reply:
Yes we do. :)
https://github.com/doctrine/lexer

Reply:
We have it since 2013. We at least want people to use this one

Reply:
Alright :+1: